### DISCUSSION TITLE
[SP-API] - How to get all ofers with asin ( More than 20 which is the maximum limit of getItemOffers )

### DISCUSSION THREAD

**Discussion by 0xs3n4** [amzn/selling-partner-api-models#3819] [General]
Currently I need to retrieve all the sellers of a specific ASIN, the getItemOffers endpoint of the product price api returns a maximum of 20, I would like all the sellers... Can anyone help me?

**Comment by kevnhowe** [comment]
looks like that endpoint does not return a pagination or nextToken property in the response. I would make a suggestion to the devs to add this feature. Seems like a half-baked implementation to only give the first 20 of however many there are.

**Reply by 0xs3n4** [reply]
i agree with you, but i believe that its is a rule of sp-api... In other endpoints has the same rule...
